### PR TITLE
Auto Dependency Updates ...

### PR DESCRIPTION
**Pull request by lguzzon**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -46,10 +46,10 @@
     "grunt-contrib-uglify": "~2.0.0",
     "grunt-contrib-watch": "~1.0.0",
     "grunt-release": "^0.14.0",
-    "grunt-simple-mocha": "^0.4.0",
+    "grunt-simple-mocha": "^0.4.1",
     "http-server": "^0.9.0",
     "jquery": "^3.1.1",
-    "qunitjs": "1.12.0",
+    "qunitjs": "2.1.0",
```
Comment (matthewp) [inline package.json]: Our tests are not compatible with qunit 2, yet.

**Comment by matthewp** [comment]
Probably if you remove the qunit change the tests might pass. Thanks for the submission!

**Comment by pYr0x** [comment]
@matthewp 
the erros are a little bit weird:
`>(node:2015) DeprecationWarning: os.tmpDir() is deprecated. Use os.tmpdir() instead.
Error calling driver.init: Error: [init({"browserName":"firefox","platform":"Windows 10","version":"50.0","name":"qunit tests - Windows 10 firefox 50.0 ","maxDuration":1800,"commandTimeout":300,"idleTimeout":300,"build":"187682033","tunnelIdentifier":"2253.4"})] The environment you requested was unavailable.`
**The environment you requested was unavailable.**

can we get better errors from sourcelab?

**Comment by matthewp** [comment]
Could be an issue with saucelabs, i'll investigate.

**Comment by matthewp** [comment]
@lguzzon Can you try rebasing this down to one commit and then pushing again?

**Pull request closed without merging**